nilearn/plotting/displays.py

```diff
@@ -1255,3 +1260,47 @@ def get_slicer(display_mode):
 def get_projector(display_mode):
     "Internal function to retrieve a projector"
     return get_create_display_fun(display_mode, PROJECTORS)
+
+
+def check_threshold(threshold, data, percentile_calculate, name):
+    """
+    Checks that the given threshold has an accepted string value
+    and returns a threshold computed on the data. Use case of data
+    is mainly from plot_connectome and plot_prob_atlas.
+
+    Parameters
+    ----------
+    threshold: a real value or a percentage in string or list of these.
```

Comment:
We are looping over the list of thresholds in ```plot_prob_atlas``` case where as ```plot_connectome``` I suppose there is no doc for list of thresholds. So, I think we are safe in both cases. Am I wrong ?